packages/base/src/mainview/mainView.tsx

```diff
@@ -37,7 +38,7 @@ import { ContextMenu } from '@lumino/widgets';
 import { Collection, MapBrowserEvent, Map as OlMap, View, getUid } from 'ol';
 //@ts-expect-error no types for ol-pmtiles
 import { PMTilesRasterSource, PMTilesVectorSource } from 'ol-pmtiles';
-import Feature, { FeatureLike } from 'ol/Feature';
+import { Feature, FeatureLike } from 'ol/Feature';
```

Comment:
```suggestion
import Feature, { FeatureLike } from 'ol/Feature';
```

This would fix the failure

Reply:
Thanks for pointing this fix out @arjxn-py